cli/install_ee/install_ee_test.go

```diff
@@ -30,9 +31,34 @@ func TestGetVersions(t *testing.T) {
 			err:    fmt.Errorf("no packages for this OS"),
 		}
 
-	inputData1 := []byte(`<a href="/enterprise/tarantool-enterprise-bundle-` +
-		`1.10.10-52-g0df29b137-r419.tar.gz">tarantool-enterprise-bundle-1.10.10-` +
-		`52-g0df29b137-r419.tar.gz</a>                                      2021-08-18 ` +
+	arch, err := util.GetArch()
+	assert.Nil(err)
+
+	osType, err := util.GetOs()
+	assert.Nil(err)
+	inputData1 := []byte(``)
+	osName := ""
+	switch osType {
+	case util.OsLinux:
+		// Bundles without specifying the architecture are all x86_64.
+		if arch == "x86_64" {
+			arch = ""
+		}
+	case util.OsMacos:
+		// Bundles without specifying the architecture are all x86_64.
+		if arch == "x86_64" {
+			osName = "-macosx-"
+			arch = "x86_64"
```

Comment:
Arch is already x86_64, why is this line needed?

Reply:
Changed